siliconcompiler/tools/klayout/klayout_utils.py

```diff
@@ -94,7 +103,9 @@ def technology(schema):
         if not os.path.isabs(layer_properties):
             layer_properties = os.path.abspath(os.path.join(os.path.dirname(tech_file),
                                                             layer_properties))
-    if schema.valid('pdk', sc_pdk, 'display', 'klayout', sc_stackup):
+    if os.path.exists(local_files['lyp']):
+        layer_properties = os.path.abspath(local_files['lyp'])
```

Comment:
any reason for inconsistency using `os.path.abspath` here vs. not for the lyt file?